azurerm/resource_arm_database_vulnerability_assessment_rule_baseline.go

```diff
@@ -0,0 +1,232 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/sql/mgmt/2017-03-01-preview/sql"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/validate"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmDatabaseVulnerabilityAssessmentRuleBaseline() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmDatabaseVulnerabilityAssessmentRuleBaselineCreateUpdate,
+		Read:   resourceArmDatabaseVulnerabilityAssessmentRuleBaselineRead,
+		Update: resourceArmDatabaseVulnerabilityAssessmentRuleBaselineCreateUpdate,
+		Delete: resourceArmDatabaseVulnerabilityAssessmentRuleBaselineDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"resource_group_name": azure.SchemaResourceGroupName(),
+
+			"server_name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.NoEmptyStrings,
```

Comment:
We can change this back to validate mssql database name

Reply:
Reverted it back